### PR TITLE
Add attributes to DCR blocks

### PR DESCRIPTION
**Pull request by joecowton1**
## What does this change?

In order to differentiate between `summary` and `keyEvent` timeline events, we need to provide block `attributes` to DCR.

### CODE REVIEW

File: common/app/model/dotcomrendering/DotcomRenderingSupportTypes.scala [hunk 1]
```diff
@@ -97,8 +98,22 @@ object Block {
...
+    }
+
+    val attributes = BlockAttributes(
+      block.attributes.pinned.getOrElse(false),
+      block.attributes.keyEvent.getOrElse(false),
+      block.attributes.summary.getOrElse(false),
+      membershipPlaceholder,
+    )
+
+    println(block.attributes)
```
Comment (abeddow91) [inline common/app/model/dotcomrendering/DotcomRenderingSupportTypes.scala:112]: nit - lets remove this

File: common/app/model/dotcomrendering/DotcomRenderingSupportTypes.scala [hunk 2]
```diff
@@ -2,9 +2,9 @@ package model.dotcomrendering
 
 import com.gu.contentapi.client.model.v1.{Block => APIBlock}
 import com.gu.contentapi.client.utils.format.ImmersiveDisplay
-import common.Edition
```
Comment (abeddow91) [inline common/app/model/dotcomrendering/DotcomRenderingSupportTypes.scala:5]: Is there a reason for taking this out?

**Review by abeddow91 — COMMENTED** [review]
Couple of small ones but otherwise all good.

File: common/app/model/dotcomrendering/DotcomRenderingSupportTypes.scala (hunk 1 above)
Comment (joecowton1) [reply common/app/model/dotcomrendering/DotcomRenderingSupportTypes.scala]: Removed

File: common/app/model/dotcomrendering/DotcomRenderingSupportTypes.scala (hunk 2 above)
Comment (joecowton1) [reply common/app/model/dotcomrendering/DotcomRenderingSupportTypes.scala]: Wasnt being used

**Review by marjisound — APPROVED** [review]
Nice one 👍

**Pull request merged**